### PR TITLE
Link to LFortran WASM compiler #46 

### PR DESCRIPTION
**Pull request by henilp105**
This PR is to add the  LFortran WASM compiler to the compilers section of the fortran-lang.org , we should reference the LFortran WASM compiler to make it more discoverable for visitors.  please refer #46 for more information.

Please provide your suggestions regarding the current description of the compiler and location of the link in the webpage.

Thanks and Regards,
Henil Shalin Panchal

CC @awvwgk @certik @Shaikh-Ubaid

### CODE REVIEW

**Comment by henilp105** [comment]
#build_preview

File: source/compilers.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ based on the NVIDIA/PGI commercial compiler.
 [LFortran](https://lfortran.org) is a modern, interactive, LLVM-based Fortran
 compiler.
 
+[LFortran WASM Compiler](https://dev.lfortran.org/) is a statically served, WASM based LFortran Compiler for running the code in the web browsers.
+
```
Comment (certik) [inline source/compilers.md:40]: How about connecting it with the previous item, something like this:
```suggestion
Also available online at [dev.lfortran.org](https://dev.lfortran.org/) using a statically served, WASM based LFortran Compiler for running code in web browsers.
```

Comment (henilp105) [reply source/compilers.md:40]: Thanks sir.

File: source/compilers.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ based on the NVIDIA/PGI commercial compiler.
 
 [LFortran](https://lfortran.org) is a modern, interactive, LLVM-based Fortran
 compiler.
-
+Also available online at [dev.lfortran.org](https://dev.lfortran.org/) using a statically served, WASM based LFortran Compiler for running code in web browsers.
```
Comment (awvwgk) [inline source/compilers.md:38]:
```suggestion
Also available online at [dev.lfortran.org](https://dev.lfortran.org/) using a statically served, WASM based LFortran Compiler for running code in web browsers.
```

**Comment by awvwgk** [comment]
#build_preview

**Comment by Shaikh-Ubaid** [comment]
It seems fine to me. Please, could someone possibly share the upcoming work to be completed in this PR?

**Comment by awvwgk** [comment]
Just let us know if you think this is ready and I will go ahead and merge.

**Comment by awvwgk** [comment]
I will refresh the preview.

#build_preview

**Comment by Shaikh-Ubaid** [comment]
The preview at https://fortran-lang.org/pr/134/en/compilers/ still seems to be the old one.

image

Please, could someone possibly share if I am using the correct link?

**Comment by henilp105** [comment]
@Shaikh-Ubaid it has been fixed, it is available for preview at: 
 https://fortran-lang.org/pr/134/en/compilers/

**Comment by certik** [comment]
I think it is all right to merge.

Our idea was to use dev.lfortran.org to host development versions of both LFortran and the website, and possibly another domain for releases, more stable; but we can do that later. Right now we have this, and it seems to be quite stable, I have not had any issues with it in the past month, so I think we can call it good enough. And if something breaks, we'll fix it.

**Comment by henilp105** [comment]
#delete_preview

**Pull request merged**